docs/learn/tutorials/versioned/samza-event-hubs-standalone.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Samza Event Hubs Connectors Example
+---
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+The [hello-samza](https://github.com/apache/samza-hello-samza) project contains an example of a high level job that consumes and produces to [Event Hubs](../../documentation/versioned/connectors/eventhubs.html) using the Zookeeper deployment model.
```

Comment:
....has an example that uses the Samza high-level API to consume and produce from EventHubs....